### DISCUSSION TITLE
question: want me to nudge you for "spring cleaning" or deal with it yourself?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#9034] [Suggest an Idea]
**What would you like Renovate to be able to do?**

We have a lot of issues still labeled `priority-5-triage`, and also some issues that have not seen any movement in a long time (1+ year).

I was going to go through all of the old stuff and nudge the Renovate maintainers, but this might be really annoying to you... 

So I'm asking here: What kind of action do you prefer? Do you want to get nudged/pinged by me, or do you prefer to do this on your own time?

**Did you already have any implementation ideas?**

I can nudge/honk at you to fix old stuff, or you can just say: "Shoo, shoo, we don't need to be honked at." 🙃

**Comment by HonkingGoose** [comment]
Sorry for opening an issue for this, I should have posted this as a discussion in the first place!
I've corrected my mistake and moved it to a discussion... 😉

**Comment by rarkins** [comment]
It's certainly fine to go through them and pick an appropriate priority label plus anything else that's missing - it doesn't cause any notifications for me. If you think there's anything specific about an issue that needs us to take a look then it's fine to ping us to get attention

**Reply by HonkingGoose** [reply]
Thank you for your quick response. I just wanted to check in that I'm not causing spam for you. 😄 

I'll go ahead and take a look at the other `priority-5-triage` issues now then. 😉
